app/views/devise/confirmations/show.html.slim

```diff
@@ -2,14 +2,14 @@
 
 
 h1.heading = t('forms.confirmation.show_hdr')
-p.italic Your password must be at least 8 characters.
+p#password_description.italic Your password must be at least 8 characters.
```

Comment:
actually, looks like that recommendation is no longer accurate (per their note at the top)